Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -3832,7 +3832,7 @@ LayoutUnit RenderBox::containingBlockLogicalHeightForPositioned(const RenderBoxM
         if (isFixedPosition && is<RenderView>(containingBlock))
             return downcast<RenderView>(containingBlock).clientLogicalHeightForFixedPosition();
 
-        const RenderBlock& cb = is<RenderBlock>(containingBlock) ? downcast<RenderBlock>(containingBlock) : *containingBlock.containingBlock();
+        const RenderBox& cb = downcast<RenderBox>(containingBlock);
```

Comment:
This change is in code that's not table specific, so I'm surprised it doesn't have ramifications outside of tables.

Reply:
Indeed! even I was surprised, could it be that it is impacting some 'skipped' tests? Is there any way - I can test skipped test locally with this patch via commandline? like ```run-webkit-tests --ignore-skip-tests xxx``` thing?

Reply:
```
    --skipped=SKIPPED   control how tests marked SKIP are run. 'default' ==
                        Skip tests unless explicitly listed on the command
                        line, 'ignore' == Run them anyway, 'only' == only run
                        the SKIP tests, 'always' == always skip, even if
                        listed on the command line.
```

Reply:
Although for some reason, this fixes the failing tests and also don't regress any WPT and non-WPT test, do you think this is right approach for this?

Reply:
I ran the tests locally across all /css/ from WPT via 'ignore' skipped tests and it seems there is nothing regressing and all failing tests are font, woff2, scrollbar related and at the same time, no unexpected pass as well. Should I run it against 'fast' directory as well.

Reply:
@alanbaradlay - should I close this PR? It is also helping and make us match "Firefox" in following [bug 245496](https://bugs.webkit.org/show_bug.cgi?id=245496)?